app/client/src/couchers/__init__.py

```diff
@@ -1 +1,2 @@
-from .client import get_client  # noqa
+from .client import get_client
+from .client import version as __version__  # noqa
```

Comment:
Where is `__version__` read?

Reply:
This allows you to do something like this:

```py3
import couchers
print(couchers.__version__)
```

It's useful so we can ask people using the package to tell us what version they're using. Sometimes people mess up their virtual environments and are actually in a different one, so it's good the package tells you the exact one it is.

Reply:
Just use `couchers.version` then, having it in thunder makes me think it is used by Python internally.

Reply:
It's pretty common to call it `__version__`, and it seems there's a PEP for it: [PEP 0396](https://www.python.org/dev/peps/pep-0396/)